bin/p2-rctl/main.go

```diff
@@ -200,9 +256,61 @@ func (r RCtl) Enable(id string) {
 }
 
 func (r RCtl) Disable(id string) {
-	err := r.rcs.Disable(fields.ID(id))
+	err := r.rcs.Disable(rc_fields.ID(id))
 	if err != nil {
 		r.logger.WithError(err).Fatalln("Could not disable replication controller in Consul")
 	}
 	r.logger.WithField("id", id).Infoln("Disabled replication controller")
 }
+
+func (r RCtl) RollingUpdate(oldID, newID string, want, need int, deletes bool) {
+	sessions := make(chan string)
+	go kp.ConsulSessionManager(api.SessionEntry{
```

Comment:
program termination - it's in a binary, so by definition it's short-lived. I didn't see the point of adding an explicit quit channel here when it would never be used. Maybe if it needed to cleanup before termination then it would be useful, but right now that's not the case.